src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/LegacyResource.kt

```diff
@@ -31,51 +29,6 @@ import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.service.NonAssociati
 class LegacyResource(
   val nonAssociationsService: NonAssociationsService,
 ) {
-  @GetMapping("/offenders/{prisonerNumber}/non-association-details")
```

Comment:
I don't see any requests to this in AppInsights for the last week 🎊

Reply:
Yes I remove the last one